tools/expressionmerge.py

```diff
@@ -35,7 +36,7 @@
     base, ext = os.path.splitext(f)
     delimiter = ';' if ext == '.csv' else '\t'
 
-    with gzip.open(f, 'rb') as csvfile:
+    with io.TextIOWrapper(io.BufferedReader(gzip.open(f))) as csvfile:
```

Comment:
Does `io.TextIOWrapper(io.BufferedReader(...)` impact performance? Can we use this for really large files as well? Maybe we should write a wrapper function for `io.TextIOWrapper(io.BufferedReader(gzip.open(f)))` and replace the `gzip.open` in all processes that use that.